src/utils/fillwidth.tsx

```diff
@@ -0,0 +1,77 @@
+import { reduce } from "lodash"
+
+const fillwidthDimensions = (items, containerWidth, gutter, targetHeight) => {
+  /**
+   * Scales an image object proportionally based on a direction (either -1 or 1)
+   * @param img a dimension object that references an artwork image
+   * @param dir the direction we need to scale an image, either -1 or 1
+   */
+  const resizeHeight = (img, dir) => {
+    img.width += img.width / img.height * dir
+    img.height += dir
```

Comment:
I would return a new object with the new sizes instead of mutating `img` directly.
so
```javascript
return {
  width: img.width + img.width / img.height * dir,
  height: img.height + dir
}
```